doc/docs/Python_User_Interface.md

```diff
@@ -1530,9 +1530,9 @@ A common point of confusion is described in [The Run Function Is Not A Loop](The
 
 **`run(step_functions..., until=condition/time)`**
 —
-Run the simulation until a certain time or condition, calling the given step functions (if any) at each timestep. The keyword argument `until` is *either* a number, in which case it is an additional time (in Meep units) to run for, *or* it is a function (of no arguments) which returns `True` when the simulation should stop.
+Run the simulation until a certain time or condition, calling the given step functions (if any) at each timestep. The keyword argument `until` is *either* a number, in which case it is an additional time (in Meep units) to run for, *or* it is a function (of no arguments) which returns `True` when the simulation should stop. `until` can also be a list of stopping conditions which may include a number and additional functions.
 
-**`run(step_functions..., until_after_sources=condition/time)`**
+**`run(step_functions..., until_after_sources=condition/time, stop_on_interrupt=False)`**
```

Comment:
the `stop_on_interrupt` keyword has been removed, no?

Reply:
Correct. Fixed.